README.md

```diff
@@ -29,7 +29,8 @@ Several people have kindly open-sourced their implementations of this algorithm
  - Andrey Kislyuk [node-complexify](https://github.com/kislyuk/node-complexify)
  - Michael Crumley [php-complexify](https://github.com/mcrumley/php-complexify/)
  - Maciej Podsiedlak [angular-complexify](https://github.com/Kraku/angular-complexify/)
-
+ - Damian Marušić (Infinum) [android-complexify](https://github.com/infinum/android-complexify)
```

Comment:
No, at the moment it does not depend on java-complexify. Android version was created few months before java. The next version will depend on java-complexify so that we do not have to maintain the algorithm on two places.